.github/workflows/build.yml

```diff
@@ -65,6 +68,7 @@ jobs:
         run: |
           find . -name '*.deb' -exec sha256sum {} \;
           # FIXME: securedrop-app-code isn't reproducible
+          # FIXME: securedrop-admin isn't reproducible
```

Comment:
IMO non-reproducibility is a blocker to move forward, but it's literally just the bytecode that's unreproducible: https://gist.github.com/legoktm/fc213ffb0dbe20385e50441aaf544a28

If you look in the current Debian packaging we already have an override to strip all the bytecode, we should do the same here.